### PR TITLE
Changing parameter names for use in Python

### PR DESCRIPTION
**Pull request by lilyclements**
Links to #47

### CODE REVIEW

**Review by dannyparsons — COMMENTED** [review]
Just one comment for a change.

File: R/climatic_summary.R [hunk 1]
```diff
@@ -47,11 +47,11 @@ climatic_summary <- function(data, date_time, station = NULL, elements,
                                     "overall"),
                              by = NULL,
                              doy = NULL, doy_first = 1, doy_last = 366, 
-                             summaries, na.rm = FALSE,
+                             summaries, na_rm = FALSE,
```
[the remark this answers is not in the document]
Comment (lilyclements) [reply R/climatic_summary.R]: Have done that now

**Comment by dannyparsons** [comment]
Thanks. Does anything in `climatic_summary` need to change to allow it to work if `elements = NULL`? I haven't looked in detail at that.

**Comment by lilyclements** [comment]
> Thanks. Does anything in `climatic_summary` need to change to allow it to work if `elements = NULL`? I haven't looked in detail at that.

On line 210, the `elements` parameter is used, and so I assume we would need to change the function for if no elements are given? Or does `all_of(elements)` still work even if there are no elements?

```
sum_data <- grp_data %>%
    dplyr::summarise(dplyr::across(dplyr::all_of(elements), 
                                   lambda_summaries, .names = names))
```

**Comment by dannyparsons** [comment]
Not sure! Would be good to check. My idea was that since `n()` as the default summary this should work without any elements, but we probably need to add a case for this. This can be done in a separate PR though.

**Pull request merged**